packages/utils/ast-utils.js

```diff
@@ -429,6 +429,62 @@ function addProperty(j, p, key, value) {
 	}
 	return;
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
```

Comment:
Also here, `p` means nothing. We could call it `propertyToAdd`

Reply:
p is fine, it's an ast node, we don't really care about the type of the node as long as we're updating its properties ( see the name of the function )